util/chplenv/chpl_3p_jemalloc_configs.py

```diff
@@ -0,0 +1,29 @@
+import os
+import sys
+
+chplenv_dir = os.path.dirname(__file__)
+sys.path.insert(0, os.path.abspath(chplenv_dir))
+
+import third_party_utils, utils
+from utils import memoize
+
+
+@memoize
+def get_uniq_cfg_path():
+    return third_party_utils.default_uniq_cfg_path()
+
+@memoize
+def get_link_args():
+    # -lpthread required to satisfy "undefined reference to `pthread_atfork"
+    # and others. Note that this only appears to be needed because we build
+    # jemalloc statically. When using a .so the -lpthread is not required.
+    # @gbt any idea why that is/anything to change or is this ok?
```

Comment:
Ugh.  Can't think of why the this would be different.  If we build a .so then does that end up with a request for libpthread.so in it?  (Think rtld will show you this, or if not that, readelf certainly can.)  And if we only build statically we don't end up with libpthread anywhere in the .la file?